api/proto/oap/collector/receiver/opentelemetry/opentelemetry.proto

```diff
@@ -23,7 +22,7 @@ message PostMetricsServiceRequest {
   // element. Intermediary nodes (such as OpenTelemetry Collector) that receive
   // data from multiple origins typically batch the data before forwarding further and
   // in that case this array will contain multiple elements.
-  repeated erda.oap.opentelemetry.ResourceMetrics resource_metrics = 1;
+  repeated erda.oap.opentelemetry.metrics.v1.ResourceMetrics resource_metrics = 1;
```

Comment:
Can we remove the prefix "erda.oap"? Maybe we can use otel proto directly.

Reply:
Done.
Introducing `proto_fetcher.sh` to solve the issue of importing external proto files.

The command remains as "make proto-go-in-local".

If users are unable to access GitHub, they can solve this by introducing a mirror. e.g.
```shell
 OTEL_PROTO_REPO_MIRROR=https://gitee.com/rocket-mirror/opentelemetry-proto.git make proto-go-in-local 
```